script/start.ts

```diff
@@ -52,7 +52,6 @@ if (process.env.NODE_ENV === 'production') {
         message,
         u(message, u(message, rendererConfig).output).publicPath
       ),
-      logLevel: 'error',
```

Comment:
This has been moved to `infrastructureLogging.level` in [webpack.development.ts](https://github.com/desktop/desktop/pull/13894/files#diff-bcf75b74c707fb68169331415b4cce71a5a14e41c81834ed8f8104df4467fce1R61-R63)